### ISSUE TITLE
train.py report duplicate steps with wide ranging loss

### ISSUE THREAD

**Issue by Nightrain** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#384]
Windows 10
Python 3.6
tensorflowgpu 1.13
NVIDIA Quadro P400
Updated the video drivers on Monday 10/28
Installed tensorflow with conda install so should have the correct CUDA and cuDNN

I am trying to build my own object detection model.  And everything "seems" to working as expected during training.  That is no errors and the console is reporting one every step.  Except it is showing duplicate steps and duplicate messages during init.  Also the loss scores are swinging wildly.  Please tell me what else I can supply to help figure this out.  Here is an example of what I seeing.  Is this a problem and I should stop training and start all over?

`INFO:tensorflow:global step 1585: loss = 1.4643 (8.599 sec/step)
INFO:tensorflow:global step 1585: loss = 1.4643 (8.599 sec/step)
INFO:tensorflow:global step 1586: loss = 8210534360940544.0000 (1.856 sec/step)
INFO:tensorflow:global step 1586: loss = 8210534360940544.0000 (1.856 sec/step)
INFO:tensorflow:global step 1587: loss = 18107264292356096.0000 (2.246 sec/step)
INFO:tensorflow:global step 1587: loss = 18107264292356096.0000 (2.246 sec/step)
INFO:tensorflow:global step 1588: loss = 2853782429892608.0000 (1.946 sec/step)
INFO:tensorflow:global step 1588: loss = 2853782429892608.0000 (1.946 sec/step)
INFO:tensorflow:global step 1589: loss = 1.6056 (5.239 sec/step)
INFO:tensorflow:global step 1589: loss = 1.6056 (5.239 sec/step)
INFO:tensorflow:global step 1590: loss = 2140216425896214528.0000 (2.372`

thanks for reading this far
cheers
bob
